common/src/chain/config/mod.rs

```diff
@@ -452,6 +456,60 @@ fn create_mainnet_genesis() -> Genesis {
     )
 }
 
+fn decode_hex<T: serialization::DecodeAll>(hex: &str) -> T {
+    let bytes = Vec::from_hex(hex).expect("Hex decoding shouldn't fail");
+    <T as serialization::DecodeAll>::decode_all(&mut bytes.as_slice())
+        .expect("Decoding shouldn't fail")
+}
+
+fn create_testnet_genesis() -> Genesis {
+    // TODO: use coin_decimals instead of a fixed value
+    const COIN: Amount = Amount::from_atoms(100000000000);
+
+    let total_amount = (COIN * 100_000_000).expect("must be valid");
+    let initial_pool_amount = (COIN * 40_000).expect("must be valid");
+    let mint_output_amount = (total_amount - initial_pool_amount).expect("must be valid");
+
+    let genesis_message = String::new();
+
+    let genesis_mint_destination = decode_hex::<PublicKey>(
+        "00027a9771bbb58170a0df36ed43e56490530f0f2f45b100c42f6f405af3ef21f54e",
+    );
+    let decommission_pub_key = decode_hex::<PublicKey>(
+        "0002ea30f3bb179c58022dcf2f4fd2c88685695f9532d6a9dd071da8d7ac1fe91a7d",
+    );
+    let staker_pub_key = decode_hex::<PublicKey>(
+        "0002884adf48b0b32ab3d66e1a8b46576dfacca5dd25b66603650de792de4dd2e483",
+    );
+
+    let vrf_pub_key = decode_hex::<VRFPublicKey>(
+        "00d4b8138be57c1bddfff397ffc72ca0453131f8df364d2b6bf4a8c6558ed81d79",
+    );
```

Comment:
It would be nice to have an unused function that generates these values so that we can generate them on the go to boot the testnet when needed.

Reply:
Was this done? If yes, please write above them how to generate these.

Reply:
I added a comment. The CLI wallet now has two new commands for this: newpublickey and getvrfpublickey.